sig-aws/README.md

```diff
@@ -0,0 +1,66 @@
+# Proposal: Spinnaker AWS Special Interest Group
+
+## Roles & Responsibilities of the SIG Leads 
+
+### SIG Lead(s):
+
+**[Clare Liguori](https://github.com/clareliguori)** \
+Principal Software Engineer \
+_AWS_
+
+**[Rob Zienert](https://github.com/robzienert)** \
+Senior Software Engineer \
+_Netflix_
+
+**[Isaac Mosquera](https://github.com/imosquera)** \
+Software Engineer  \
+_Armory_
+
+
+### SIG Lead Responsibility
+
+1. Coordinate communication.
+2. Set meeting dates and objectives for the coming year.
+3. Curate agenda for each meeting.
+4. Raise awareness and invite appropriate participants.
+5. Encourage and foster an inclusive environment.
+6. Facilitate the identification and development of resources (people or funding) for support of AWS within Spinnaker.
+
+
+### Meeting Cadence
+ - 1 hour, monthly, on **[Google Hangouts](https://meet.google.com/yfp-ekod-sxj)**
+ - Third Wednesday of every month at 3PM PST
+ - [AWS SIG Notes](https://docs.google.com/document/d/1TB7dSQDTM9jFBsttuevxOsP6MGQ3-z0wrYMIGxYxhYQ/edit)
+
+# Responsibilities
+
+## Description 
+
+AWS SIG is responsible for the creation and maintenance of subprojects & subservices necessary to integrate AWS services for the operation and management of Spinnaker on AWS as well as using AWS as a deployment target. 
+
+AWS SIG also acts as a forum for discussions and feature demos for Spinnaker users on AWS users/developers to raise their feature requests and support issues. SIG leads in collaboration with SIG members will develop, maintain and prioritize a backlog of features.  This prioritization will be published as the AWS SIG Roadmap.
+
+
+## Scope of Work
+
+Spinnaker integrations specific to AWS including:
+
+*   Integrations, interfaces, libraries and extension points for all low-level AWS modules such as IAM, storage, networking, load balancers, registry, security, monitoring/logging at the instance or container level
+*   Development of Spinnaker APIs to provide software delivery best practices for AWS computational services which might include, but not limited to, the following:
```

Comment:
Nit: we typically refer to "AWS compute services", not computational: https://docs.aws.amazon.com/whitepapers/latest/aws-overview/compute-services.html